Python/getargs.c

```diff
@@ -2666,7 +2679,6 @@ skipitem(const char **p_format, va_list *p_va, int flags)
     case 'D': /* complex double */
     case 'c': /* char */
     case 'C': /* unicode char */
-    case 'p': /* boolean predicate */
```

Comment:
Why not put new code right here?:)

Reply:
It would break the fallthrough from the other cases, unfortunately

Reply:
Is this conversation resolved?

Reply:
I think so.